addons/web_editor/views/snippets.xml

```diff
@@ -403,10 +403,10 @@
         </we-row>
 
         <we-button-group string="Width">
-            <we-button data-set-width="" title="Resize Auto">Auto</we-button>
-            <we-button data-set-width="25%" title="Resize Quarter">25%</we-button>
-            <we-button data-set-width="50%" title="Resize Half">50%</we-button>
-            <we-button data-set-width="100%" title="Resize Full">100%</we-button>
+            <we-button data-select-class="" title="Resize Auto">Auto</we-button>
+            <we-button data-select-class="w-25" title="Resize Quarter">25%</we-button>
+            <we-button data-select-class="w-50" title="Resize Half">50%</we-button>
+            <we-button data-select-class="w-100" title="Resize Full">100%</we-button>
```

Comment:
Not a bad idea but I'd rather keep the inline style in that case, otherwise the old inline style on people images will not be able to be removed. I prefer not to change the behavior from previous versions for this. You're right though that there is no point recoding an option method for that, you can just use `select-style`. I already reported it to the editor team but well...